src/gdcmodels/esmodels/__init__.py

```diff
@@ -0,0 +1,86 @@
+from typing import Any, Dict, KeysView, Literal, Mapping, Sequence, Union, overload
```

Comment:
nit: can the contents of this module be moved to a separate module that is not an `__init__` file